workspace-service/Makefile

```
@@ -27,19 +29,25 @@ docker-push: ## Push Docker image
 	docker push $(image)
 	docker push $(image):$(tag)
 
+.PHONY: run-local
```

```suggestion
.PHONY: watch-local
```
oops. Managed to get them out of sync with my previous suggestion.